DEVELOPERS.md

```diff
@@ -72,9 +77,9 @@ Before you start a development server, you must choose if you want to work on th
 
 To debug code, and to see changes in real time, it is often useful to have a local HTTP server. Click one of the three links below to choose which development server you want to start.
 
-- [Supabase Website](#Supabase-Website-Development-Server)
-- [Supabase Docs](#Supabase-Docs-Development-Server)
-- [Supabase Studio](#Supabase-Studio-Development-Server)
+- [Supabase Website](###Supabase-Website-Development-Server)
+- [Supabase Docs](###Supabase-Docs-Development-Server)
+- [Supabase Studio](###Supabase-Studio-Development-Server)
```

Comment:
Any reason why change this?

Reply:
Tbh, I don't know 😬 maybe while copy-pasting it might have changed. Sorry

Reply:
I have re-added those

Reply:
These changes break the links and need to be undone.